conda_build/config.py

```diff
@@ -58,7 +99,7 @@ def set_invocation_time():
 no_rewrite_stdout_env_default = "false"
 ignore_verify_codes_default = []
 exit_on_verify_error_default = False
-conda_pkg_format_default = None
+conda_pkg_format_default = CondaPkgFormat.V1
```

Comment:
```suggestion
conda_pkg_format_default = CondaPkgFormat.V2
```

Reply:
I think this change should be postponed to 25.1.